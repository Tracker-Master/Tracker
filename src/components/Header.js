import React from 'react';
import { Link } from 'react-router-dom';

import {
  StyledHeader,
  HeaderContainer,
  Logo,
  Nav,
} from 'Styles/components/HeaderStyles';

const Header = () => {
  return (
    <StyledHeader>
      <HeaderContainer>
        <Nav>
          <ul>
            <li>
              <img
                src="https://assets-juanjosemayorga-website.s3.amazonaws.com/tracker-project-s3/menu-24px.svg"
                alt="burger-menu"
              />
            </li>
          </ul>
        </Nav>
        <Link to="/">
          <Logo>
            <img
              src="https://assets-juanjosemayorga-website.s3.amazonaws.com/tracker-project-s3/tracker-logo.svg"
              alt="Logo"
            />
          </Logo>
        </Link>
      </HeaderContainer>
    </StyledHeader>
  );
};

export default Header;