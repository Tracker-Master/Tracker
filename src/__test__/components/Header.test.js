import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import { StaticRouter } from 'react-router';
import Header from 'Components/Header';

describe('<Header />', () => {
  const header = shallow(
    <StaticRouter>
      <Header />
    </StaticRouter>
  );

  test('Header rendering', () => {
    expect(header.length).toBe(1);
  });

  test('Header snapshot', () => {
    const wrapper = create(
      <StaticRouter>
        <Header />
      </StaticRouter>
    );
    expect(wrapper).toMatchSnapshot();
  });
  
});