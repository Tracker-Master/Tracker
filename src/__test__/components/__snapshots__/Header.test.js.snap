// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Header snapshot 1`] = `
.c0 {
  display: inherit;
  background-color: #673AB7;
  color: #fff;
}

.c1 {
  display: inherit;
  grid-template: 1fr/1fr 1fr;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0 auto;
  padding: 0 5%;
  width: 100%;
  height: inherit;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  height: 50px;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  height: inherit;
}

.c2 ul {
  display: inherit;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  width: 100%;
}

.c2 ul li {
  list-style: none;
  cursor: pointer;
}

.c2 ul li img {
  width: 2rem;
  height: inherit;
}

@media (min-width:768px) {

}

@media (min-width:1024px) {

}

@media (min-width:768px) {

}

@media (min-width:1024px) {
  .c1 {
    max-width: 1024px;
    padding: 0;
  }
}

@media (min-width:768px) {

}

@media (min-width:1024px) {

}

@media (min-width:768px) {
  .c2 ul {
    width: 50%;
  }
}

@media (min-width:1024px) {
  .c2 ul {
    width: 50%;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <nav
      className="c2"
    >
      <ul>
        <li>
          <img
            alt="burger-menu"
            src=""
          />
        </li>
      </ul>
    </nav>
    <a
      href="/"
      onClick={[Function]}
    >
      <picture
        className="c3"
      >
        <img
          alt="Logo"
          src="https://assets-juanjosemayorga-website.s3.amazonaws.com/tracker-project-s3/tracker-logo.svg"
        />
      </picture>
    </a>
  </div>
</div>
`;
